assets/stylesheets/bootstrap/_panels.scss

```diff
@@ -5,23 +5,37 @@
 
 // Base class
 .panel {
-  margin-bottom: $line-height-computed;
+  margin-top: 0.5rem;
   background-color: $panel-bg;
-  border: 1px solid transparent;
-  border-radius: $panel-border-radius;
+  border-left: 3px solid $panel-primary-border;
+  border-bottom: 1px solid $panel-footer-bg;
+
+  &.odd {
+    background-color: $white;
+  }
+
+  .flexbox & {
+    & .panel-body {
+      display: flex;
+      flex-wrap: wrap;
+      align-items: center;
+    }
+  }
 }
 
 // Panel contents
 .panel-body {
+  color: $black;
```

Comment:
This overrides secondary nested text, see the example below (inner MAX text is black rather than grey):

![panel](https://cloud.githubusercontent.com/assets/557957/18636934/93abca44-7e82-11e6-84ca-57a752409116.png)

Maybe this should only apply to `span` and `p` type elements?